apps/desktop/src/components/v3/CommitView.svelte

```diff
@@ -164,10 +156,25 @@
 				/>
 			</Drawer>
 		{:else}
-			<Drawer projectId={env.projectId} stackId={env.stackId} splitView>
+			<Drawer projectId={env.projectId} stackId={env.stackId}>
 				{#snippet header()}
 					<div class="commit-view__header text-13">
-						<CommitLine width={24} {commit} />
+						{#if isLocalAndRemoteCommit(commit)}
+							<CommitLine
+								commitStatus={commit.state.type}
+								diverged={commit.state.type === 'LocalAndRemote' &&
+									commit.id !== commit.state.subject}
```

Comment:
There are at least 3 places where we do this check, should be a helper function.